tests/python/openassetio/hostApi/test_managerfactory.py

```diff
@@ -127,6 +128,91 @@ def test_when_has_implementations_then_reports_implementation_details(
         assert actual == expected
 
 
+class Test_ManagerFactory_kDefaultManagerConfigEnvVarName:
+    def test_has_expected_value(self):
+        assert ManagerFactory.kDefaultManagerConfigEnvVarName == "OPENASSETIO_DEFAULT_CONFIG"
+
+
+class Test_ManagerFactory_defaultManagerForInterface:
+    def test_when_var_not_set_then_returns_none(
+        self, mock_manager_implementation_factory, mock_host_interface, mock_logger
+    ):
+        assert (
+            ManagerFactory.defaultManagerForInterface(
+                mock_host_interface, mock_manager_implementation_factory, mock_logger
+            )
+            is None
+        )
+
+    def test_when_var_set_to_non_existent_path_then_runtime_error_raised(
+        self,
+        env_with_non_existent_manager_config,  # pylint: disable=unused-argument
+        mock_manager_implementation_factory,
+        mock_host_interface,
+        mock_logger,
+    ):
+        with pytest.raises(RuntimeError) as exc:
+            ManagerFactory.defaultManagerForInterface(
+                mock_host_interface, mock_manager_implementation_factory, mock_logger
+            )
+        assert (
+            str(exc.value)
+            == "Could not load default manager config from 'i/do/not/exist', file does not exist."
+        )
+
+    def test_when_var_set_to_invalid_content_then_runtime_error_raised(
+        self,
+        env_with_invalid_manager_config,  # pylint: disable=unused-argument
+        mock_manager_implementation_factory,
+        mock_host_interface,
+        mock_logger,
+    ):
+        with pytest.raises(RuntimeError):
+            ManagerFactory.defaultManagerForInterface(
+                mock_host_interface, mock_manager_implementation_factory, mock_logger
+            )
+
+    def test_when_var_set_to_valid_path_then_expected_manager_returned_with_expected_settings(
+        self,
+        env_with_test_manager_config,  # pylint: disable=unused-argument
+        mock_manager_implementation_factory,
+        mock_host_interface,
+        mock_logger,
+        create_mock_manager_interface,
+    ):
+
+        expected_host_identifier = "a.host"
+        mock_host_interface.mock.identifier.return_value = expected_host_identifier
+
+        expected_settings = {
+            "a_string": "Hello 🐈",
+            "a_float": 3.141579,
+            "a_bool": False,
+            "a_int": 42,
+        }
+
+        mock_manager_interface = create_mock_manager_interface()
+        mock_manager_interface.mock.identifier.return_value = "identifier.from.toml.file"
+
+        # Use a side-effect to check host_session as this is constructed
+        # privately by the factory.
+        def check_initialization(settings, host_session):
+            assert settings == expected_settings
```

Comment:
[consider] If I delete this assertion, then blank out the `default_manager.toml` (the intention was to check if any exceptions were raised), then the tests still pass.  We should `assert_called_once_with` on `instantiate`.

Reply:
Improved test coverage [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/608/commits/4b2128fd95db54ce809070d6d406b48ad184a555).

Reply:
Looks good.  Just a quick question.  A blank .toml file (or equivalently, blank manager identifier) is valid from the perspective of the `ManagerFactory`, but am I correct in thinking that's fine because it's up to the `ManagerImplementationFactory` how to interpret it (and potentially raise an exception)?

Reply:
A good Q. off-line we pondered that we'd leave as-is for now, and just surface the error from the underlying implementation factory.